src/middlewared/middlewared/plugins/pool_/replace_disk.py

```diff
@@ -66,8 +66,6 @@ async def replace(self, job, oid, options):
                 # If the disk we are replacing is still available, remove it from swap as well
                 swap_disks.append(from_disk)
```

Comment:
Remove this line (and `swap_disks` variable altogether), but keep `from_disk` as it is being wiped later